tests/trainer/test_predict.py

```diff
@@ -28,6 +33,21 @@ def _assert_predict_events_called_expected_number_of_times(
         assert expected == actual, f'Event {event} expected to be called {expected} times, but instead it was called {actual} times'
 
 
+class PredictionSaver(Callback):
+
+    def __init__(self, folder: str):
+        self.folder = folder
+        os.makedirs(self.folder, exist_ok=True)
+
+    def predict_batch_end(self, state: State, logger: Logger) -> None:
```

Comment:
comment: there might be a situation, where the logger has a `log_table` function that we call here